### PR TITLE
Integrate assist onboarding & fixes

### PR DESCRIPTION
**Pull request by liamaharon**
- [x] Remove the `lazyLoadWeb3` action entirely as all the `ethereum.enable()`-like checks are done when we call `onboardAssist()`.
- [x] Configure the built-in "Install MetaMask" banner to only show if the user exits the assist onboarding without having given access to their accounts.
- [x] Show built-in notifications on mobile
- [x] Remove hardcoded gasPrice
- [x] Finish migrating to web3
- [x] Fix final mobile issues
- [x] Update assist

Let me know if you have any questions!

### CODE REVIEW

**Comment by andygray** [comment]
Seems to be setup for rinkeby if localhost. We can adjust this.
image

**Comment by andygray** [comment]
Looks good!

**Comment by liamaharon** [comment]
Hey @andygray , yeah I've set up the onboarding to setup the user on Rinkeby in when the app is running in development, and mainnet when the app is running in production. 

You can change the network to use for each environment in the `utils.js` file

File: src/utils.js [hunk 1]
```diff
@@ -7,15 +7,14 @@ const API_CONFIG = {
...
-        default:
-            return API_CONFIG.live;
-    }
-};
+const live = window.location.hostname !== "localhost" && window.location.hostname !== "127.0.0.1"
+
+// if live use mainnet, otherwise use rinkeby
+const DEV_NETWORK_ID = 4;
+const PROD_NETWORK_ID = 1;
+const NETWORK_ID = live ? PROD_NETWORK_ID : DEV_NETWORK_ID;
```
Comment (jamesmorgan) [inline src/utils.js:15]: I think we will tweak this once its in as we'd like to test on live from local as well, no biggy we can change this

**Pull request merged**